### DISCUSSION TITLE
SideSetsBetweenSubdomainsGenerator

### DISCUSSION THREAD

**Discussion by joe61vette** [idaholab/moose#18474] [Q&A Meshing] [answered]
Hi:  using mesh generators to simplify a larger mesh.  Would like to place a sideset between one large block 'Pebble_Bed' and two adjacent smaller blocks 'Chute_Inlet' and 'Inlet_Annulus'.  I have tried several sets of commands like:

  [bottom_ss]
    type = SideSetsBetweenSubdomainsGenerator
    input = top_ss
    new_boundary = 'inlet'
    primary_block = 'Pebble_Bed Pebble_Bed'
    paired_block = 'Chute_Inlet Inlet_Annulus'
  []

This does generate a sideset, but only between Pebble_Bed and Inlet_Annulus.  Is it possible to do this?

Thanks,
  Joe

**Comment by GiudGiud** [comment, marked as answer]
Hi Joe

Could you please add show_info to the Mesh block and paste the output here?
I have never done it with a single generator, but probably should work. Let s look at the log. 

Guillaume

**Reply by joe61vette** [reply]
Hi Guillaume:  just after the sidesetgenerator block, I delete all of the blocks except Pebble_Bed.  So, there is a lot of stuff in the log but I don't see anything helpful.  Here it is:

SideSetsBetweenSubdomainsGenerator 'bottom_ss':  Mesh Information:
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   elem_dimensions()={3}
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   spatial_dimension()=3
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   n_nodes()=5083
SideSetsBetweenSubdomainsGenerator 'bottom_ss':     n_local_nodes()=1154
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   n_elem()=5704
SideSetsBetweenSubdomainsGenerator 'bottom_ss':     n_local_elem()=817
SideSetsBetweenSubdomainsGenerator 'bottom_ss':     n_active_elem()=5704
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   n_subdomains()=14
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   n_partitions()=6
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   n_processors()=6
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   n_threads()=1
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   processor_id()=0
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   is_prepared()=true
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   is_replicated()=true
SideSetsBetweenSubdomainsGenerator 'bottom_ss': 
SideSetsBetweenSubdomainsGenerator 'bottom_ss':  Mesh Bounding Box:
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Minimum: (x,y,z)=(-6.33924e-18, -0.496933,        0)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Maximum: (x,y,z)=(    1.92, 0.496933,      5.1)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Delta:   (x,y,z)=(    1.92, 0.993865,      5.1)
SideSetsBetweenSubdomainsGenerator 'bottom_ss': 
SideSetsBetweenSubdomainsGenerator 'bottom_ss':  Mesh Element Type(s):
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   HEX8, PRISM6
SideSetsBetweenSubdomainsGenerator 'bottom_ss': 
SideSetsBetweenSubdomainsGenerator 'bottom_ss':  Mesh Nodesets:
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   None
SideSetsBetweenSubdomainsGenerator 'bottom_ss': 
SideSetsBetweenSubdomainsGenerator 'bottom_ss':  Mesh Sidesets:
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Sideset 1 (inlet_chute), 8 sides (QUAD4), 8 elems (HEX8), 14 nodes
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Side volume: 0.0104421
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box minimum: (x,y,z)=(-6.33924e-18, -0.0517638,        0)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box maximum: (x,y,z)=(     0.2, 0.0517638,        0)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box delta: (x,y,z)=(     0.2, 0.103528,        0)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Sideset 2 (inlet_annulus), 32 sides (QUAD4), 32 elems (HEX8), 45 nodes
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Side volume: 0.365473
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box minimum: (x,y,z)=(0.193185, -0.310583,        0)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box maximum: (x,y,z)=(     1.2, 0.310583,        0)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box delta: (x,y,z)=( 1.00681, 0.621166,        0)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Sideset 3 (defueling_chute), 8 sides (QUAD4), 8 elems (HEX8), 15 nodes
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Side volume: 0.0946315
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box minimum: (x,y,z)=(0.579555, -0.219996,      5.1)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box maximum: (x,y,z)=(    0.85, 0.219996,      5.1)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box delta: (x,y,z)=(0.270445, 0.439992,        0)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Sideset 4 (outlet_plenum), 12 sides (QUAD4), 12 elems (HEX8), 20 nodes
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Side volume: 0.187958
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box minimum: (x,y,z)=( 1.01422, -0.349406,     4.65)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box maximum: (x,y,z)=(    1.35, 0.349406,     4.65)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box delta: (x,y,z)=(0.335778, 0.698811,        0)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Sideset 5 (outer_wall), 136 sides (QUAD4), 136 elems (HEX8), 175 nodes
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Side volume: 5.12342
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box minimum: (x,y,z)=( 1.85458, -0.496933,        0)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box maximum: (x,y,z)=(    1.92, 0.496933,      5.1)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box delta: (x,y,z)=(0.0654224, 0.993865,      5.1)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Sideset 6 (bed_outside_wall), 80 sides (QUAD4), 80 elems (HEX8), 105 nodes
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Side volume: 1.50689
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box minimum: (x,y,z)=( 1.15911, -0.310583,     1.65)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box maximum: (x,y,z)=(     1.2, 0.310583,     4.05)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box delta: (x,y,z)=(0.040889, 0.621166,      2.4)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Sideset 7 (bed_top_wall), 32 sides (QUAD4), 32 elems (HEX8), 44 nodes
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Side volume: 0.328606
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box minimum: (x,y,z)=(1.90177e-17, -0.155291,     4.05)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box maximum: (x,y,z)=(     0.6, 0.155291,      5.1)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box delta: (x,y,z)=(     0.6, 0.310583,     1.05)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Sideset 8 (chute_outside_wall), 12 sides (QUAD4), 12 elems (HEX8), 20 nodes
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Side volume: 0.311319
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box minimum: (x,y,z)=(0.821037, -0.219996,      4.4)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box maximum: (x,y,z)=(    0.85, 0.219996,      5.1)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box delta: (x,y,z)=(0.028963, 0.439992,      0.7)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Sideset 9 (outlet), 12 sides (QUAD4), 12 elems (HEX8), 20 nodes
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Side volume: 0.265174
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box minimum: (x,y,z)=(0.821037, -0.310583,     4.05)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box maximum: (x,y,z)=(     1.2, 0.310583,      4.4)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box delta: (x,y,z)=(0.378963, 0.621166,     0.35)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Sideset 10 (inlet), 32 sides (QUAD4), 32 elems (HEX8), 45 nodes
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Side volume: 0.480094
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box minimum: (x,y,z)=(0.193185, -0.310583,      0.8)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box maximum: (x,y,z)=(     1.2, 0.310583,     1.65)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box delta: (x,y,z)=( 1.00681, 0.621166,     0.85)
SideSetsBetweenSubdomainsGenerator 'bottom_ss': 
SideSetsBetweenSubdomainsGenerator 'bottom_ss':  Mesh Edgesets:
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   None
SideSetsBetweenSubdomainsGenerator 'bottom_ss': 
SideSetsBetweenSubdomainsGenerator 'bottom_ss':  Mesh Subdomains:
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Subdomain 100 (Core_Barrel): 136 elems (HEX8, 136 active), 335 active nodes
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Volume: 0.144986
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box minimum: (x,y,z)=( 1.73867, -0.473639,        0)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box maximum: (x,y,z)=(    1.83, 0.473639,      5.1)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box delta: (x,y,z)=(0.0913335, 0.947278,      5.1)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Subdomain 200 (Outer_Reflector): 3196 elems (PRISM6, 3196 active), 1966 active nodes
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Volume: 1.85115
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box minimum: (x,y,z)=(   1.304, -0.465874,        0)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box maximum: (x,y,z)=(     1.8, 0.465874,      5.1)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box delta: (x,y,z)=(   0.496, 0.931749,      5.1)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Subdomain 300 (Inner_Reflector): 244 elems (HEX8, 244 active), 470 active nodes
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Volume: 0.434946
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box minimum: (x,y,z)=( 1.15911, -0.349406,        0)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box maximum: (x,y,z)=(    1.35, 0.349406,     4.65)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box delta: (x,y,z)=(0.190889, 0.698811,     4.65)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Subdomain 400 (Chute_Reflector): 36 elems (HEX8, 36 active), 80 active nodes
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Volume: 0.0566049
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box minimum: (x,y,z)=(0.821037, -0.27176,      4.4)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box maximum: (x,y,z)=(    1.05,  0.27176,      5.1)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box delta: (x,y,z)=(0.228963,  0.54352,      0.7)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Subdomain 500 (Center_Reflector): 60 elems (HEX8, 60 active), 116 active nodes
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Volume: 0.076748
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box minimum: (x,y,z)=(1.90177e-17, -0.155291,     4.05)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box maximum: (x,y,z)=(     0.6, 0.155291,      5.1)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box delta: (x,y,z)=(     0.6, 0.310583,     1.05)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Subdomain 700 (Vessel_Wall): 272 elems (HEX8, 272 active), 525 active nodes
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Volume: 0.301954
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box minimum: (x,y,z)=( 1.79662, -0.496933,        0)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box maximum: (x,y,z)=(    1.92, 0.496933,      5.1)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box delta: (x,y,z)=(0.123378, 0.993865,      5.1)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Subdomain 1000 (Pebble_Bed): 800 elems (HEX8, 800 active), 1134 active nodes
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Volume: 1.11538
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box minimum: (x,y,z)=(2.17911e-18, -0.310583,      0.8)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box maximum: (x,y,z)=(     1.2, 0.310583,      4.4)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box delta: (x,y,z)=(     1.2, 0.621166,      3.6)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Subdomain 1100 (Inlet_Chute): 64 elems (HEX8, 64 active), 126 active nodes
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Volume: 0.00835368
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box minimum: (x,y,z)=(-6.33924e-18, -0.0517638,        0)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box maximum: (x,y,z)=(     0.2, 0.0517638,      0.8)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box delta: (x,y,z)=(     0.2, 0.103528,      0.8)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Subdomain 1200 (Defueling_Chute): 24 elems (HEX8, 24 active), 60 active nodes
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Volume: 0.066242
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box minimum: (x,y,z)=(0.579555, -0.219996,      4.4)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box maximum: (x,y,z)=(    0.85, 0.219996,      5.1)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box delta: (x,y,z)=(0.270445, 0.439992,      0.7)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Subdomain 2000 (Inlet_Annulus): 256 elems (HEX8, 256 active), 405 active nodes
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Volume: 0.484687
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box minimum: (x,y,z)=(0.193185, -0.310583,        0)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box maximum: (x,y,z)=(     1.2, 0.310583,     1.65)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box delta: (x,y,z)=( 1.00681, 0.621166,     1.65)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Subdomain 3000 (Outlet_Annulus): 36 elems (HEX8, 36 active), 80 active nodes
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Volume: 0.0988736
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box minimum: (x,y,z)=(0.821037, -0.349406,     4.05)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box maximum: (x,y,z)=(    1.35, 0.349406,     4.65)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box delta: (x,y,z)=(0.528963, 0.698811,      0.6)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Subdomain 4000 (Control_Rod): 408 elems (HEX8, 408 active), 588 active nodes
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Volume: 0.0360624
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box minimum: (x,y,z)=(   1.525,    -0.05,        0)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box maximum: (x,y,z)=(   1.625,     0.05,      5.1)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box delta: (x,y,z)=(     0.1,      0.1,      5.1)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Subdomain 5000 (Outlet_Plenum): 36 elems (HEX8, 36 active), 80 active nodes
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Volume: 0.084581
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box minimum: (x,y,z)=( 1.01422, -0.349406,     4.65)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box maximum: (x,y,z)=(    1.35, 0.349406,      5.1)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box delta: (x,y,z)=(0.335778, 0.698811,     0.45)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Subdomain 6000 (Downcomer): 136 elems (HEX8, 136 active), 350 active nodes
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Volume: 0.147382
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box minimum: (x,y,z)=( 1.76764, -0.481403,        0)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box maximum: (x,y,z)=(    1.86, 0.481403,      5.1)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':    Bounding box delta: (x,y,z)=(0.0923557, 0.962807,      5.1)
SideSetsBetweenSubdomainsGenerator 'bottom_ss':   Global mesh volume = 4.90795
Writing Exodus

**Reply by GiudGiud** [reply]
solved over email. Block syntax is valid
